framework/auth/__init__.py

```diff
@@ -117,20 +117,17 @@ def verify_two_factor(user_id, two_factor_code):
     return response
 
 
-def login(username, password):
+def login(email, password):
```

Comment:
The purpose of the `username` field has changed - it's now the primary email for the account. It's possible for the primary email to be unconfirmed, as is the case for an unregistered user or a user who has not yet completed email verification.